### ISSUE TITLE
[BUG] - No module named 'TikTokApi.browser_utilities'

### ISSUE THREAD

**Issue by klauria22** [davidteather/TikTok-Api#709]
**Describe the bug**

When I try to create an instance of the api, it gives an error saying ModuleNotFoundError: No module named 'TikTokApi.browser_utilities'

**The buggy code**

```
api = TikTokApi.get_instance(custom_verifyFp=verifyFp, use_test_endpoints=True)
```

**Error Trace (if any)**

Put the error trace below if there's any error thrown.
```

Traceback (most recent call last):
  File "C:/Users/kevin/PycharmProjects/tiktokcompilations/main.py", line 17, in <module>
    api = TikTokApi.get_instance(custom_verifyFp=verifyFp, use_test_endpoints=True)
  File "C:\Users\kevin\tiktokcompilations\lib\site-packages\TikTokApi\tiktok.py", line 151, in get_instance
    TikTokApi(**kwargs)
  File "C:\Users\kevin\tiktokcompilations\lib\site-packages\TikTokApi\tiktok.py", line 57, in __init__
    from .browser_utilities.browser import browser
ModuleNotFoundError: No module named 'TikTokApi.browser_utilities'
```

**Desktop (please complete the following information):**
 - OS: Windows 10
 - TikTokApi Version 4.0.3

**Comment by dylancaponi** [comment]
I experienced the same issue on macOS 10.14.6.

This is not an issue with 4.0.2

**Comment by brianbruno** [comment]
Same issue here, Ubuntu 20.04.

TikTokApi Version 4.0.3

**Comment by davidteather** [comment]
Fixed by #710 V4.0.4 upgrade with `pip install TikTokApi --upgrade`
